specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/playbookConfigurations.json

```diff
@@ -0,0 +1,511 @@
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/playbookConfigurations": {
+      "get": {
+        "x-ms-examples": {
+          "Get playbook configurations of a single resource group": {
+            "$ref": "./examples/PlaybookConfigurations/GetPlaybookConfigurationsResourceGroup_example.json"
+          }
+        },
+        "tags": [ "PlaybookConfigurations" ],
+        "description": "Get all playbook configurations in a resource group.",
+        "operationId": "PlaybookConfigurations_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PlaybookConfigurationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/playbookConfigurations/{playbookConfigurationName}": {
+      "get": {
+        "x-ms-examples": {
+          "Get a single playbook configuration details": {
+            "$ref": "./examples/PlaybookConfigurations/GetPlaybookConfigurationResourceGroup_example.json"
+          }
+        },
+        "tags": [ "PlaybookConfigurations" ],
+        "description": "Get the details of a playbook configuration.",
+        "operationId": "PlaybookConfigurations_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/PlaybookConfigurationName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PlaybookConfiguration"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
>put [](start = 7, length = 3)

Is this going to be a tracked resource in ARM. If yes, then you will also need to support PATCH with ability to patch resource tags at the minimum.

Reply:
Done. Added patch api and example (for tags only)